linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/ansible/shared.yml

```
@@ -14,6 +14,16 @@
 - name: Verify that Interactive Boot is Disabled (runtime)
   command: /usr/bin/grub2-editenv - unset systemd.confirm_spawn
 
+- name: Veify GRUB_DISABLE_RECOVERY=true
```

```suggestion
- name: Verify GRUB_DISABLE_RECOVERY=true
```
Does it make sense to keep this only in SLE specific block? The OVAL has this requirement for all applicable products.